components/Footer.tsx

```diff
@@ -6,7 +6,6 @@ const FooterLinks = () => {
   const links = [
     { title: "About", path: "/about" },
     { title: "Explore the patterns", path: "/patterns" },
-    { title: "Submit an entry", path: "/submit" },
```

Comment:
these links get mapped to Next's internal `<Link>` so I don't think it makes sense to have an external one in the list!